roles/puppet_agent/defaults/main.yml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Our typical best practice has been to scope all variables here with the role name. I think switching them from `puppet_` to `puppet_agent` would be good.

Reply:
Good point. This is a leftover because the role started as `puppet` but I decided `puppet_agent` was more appropriate.